backend/src/controller/seed.rs

```diff
@@ -132,3 +135,19 @@ pub async fn edit_by_id(
     let response = service::seed::edit(*id, user_info.id, edit_seed_json.0, &app_data).await?;
     Ok(HttpResponse::Accepted().json(response))
 }
+
+/// Endpoint archiving/unarchiving a [`Seed`](crate::model::entity::Seed).
+///
```

Comment:
Mention that the timestamp will be recorded. (which btw. is a very nice solution)